chia/server/start_wallet.py

```diff
@@ -56,8 +57,10 @@ def create_wallet_service(
     fnp = service_config.get("full_node_peer")
 
     if fnp:
-        connect_peers = [PeerInfo(fnp["host"], fnp["port"])]
-        node.full_node_peer = PeerInfo(fnp["host"], fnp["port"])
+        node.full_node_peer = PeerInfo(
+            str(get_host_addr(fnp["host"], prefer_ipv6=config.get("prefer_ipv6", False))), fnp["port"]
```

Comment:
same comment here. it looks suspicious to do `get_host_addr()` witoout `await`

Reply:
https://github.com/Chia-Network/chia-blockchain/pull/14069#discussion_r1047931543